app/controllers/admin/dashboards_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Admin
+  class DashboardsController < ApplicationController
+    before_filter :authenticate_user!
+    before_filter do
```

Comment:
it would also be best to define a user instance method `admin?` 

https://pragprog.com/articles/tell-dont-ask